platform/windows/display_server_windows.cpp

```diff
@@ -341,6 +342,67 @@ String DisplayServerWindows::clipboard_get() const {
 	return ret;
 }
 
+Ref<Image> DisplayServerWindows::clipboard_get_image() const {
+	Ref<Image> image;
+	if (!windows.has(last_focused_window)) {
+		return image; // No focused window?
+	}
+	if (!OpenClipboard(windows[last_focused_window].hWnd)) {
+		ERR_FAIL_V_MSG(image, "Unable to open clipboard.");
+	}
+	UINT png_format = RegisterClipboardFormatA("PNG");
+	if (png_format && IsClipboardFormatAvailable(png_format)) {
+		HANDLE png_handle = GetClipboardData(png_format);
+		if (png_handle) {
+			size_t png_size = GlobalSize(png_handle);
+			uint8_t *png_data = (uint8_t *)GlobalLock(png_handle);
+			image.instantiate();
+
+			PNGDriverCommon::png_to_image(png_data, png_size, false, image);
+
+			GlobalUnlock(png_handle);
+		}
+	} else if (IsClipboardFormatAvailable(CF_DIB)) {
+		HGLOBAL mem = GetClipboardData(CF_DIB);
+		if (mem != NULL) {
+			BITMAPINFO *ptr = static_cast<BITMAPINFO *>(GlobalLock(mem));
+
+			if (ptr != NULL) {
+				BITMAPINFOHEADER *info = &ptr->bmiHeader;
+				PackedByteArray pba;
+
```

Comment:
Consider using `resize` and `write` instead of `append` since it can be slow due to multiple reallocation.
```suggestion
	pba.resize(info->biHeight * info->biWidth * 4);
```
and 
```
	pba.write[i] = rgbquad->rgbRed;
	pba.write[i + 1] = rgbquad->rgbGreen;
	pba.write[i + 2] = rgbquad->rgbBlue;
	pba.write[i + 3] = rgbquad->rgbReserved;
```